test/tests/functional/pbs_job_array.py

```diff
@@ -610,3 +610,32 @@ def test_job_id_duplicate_server_restart(self):
             self.server.submit(j)
         except PbsSubmitError as e:
             raise self.failureException("Failed to submit job: " + str(e.msg))
+
+    def test_expired_subjobs_not_reported(self):
+        """
+        Test if a subjob is finished and moves to expired state,
+        it is not reported to scheduler in the next scheduling
+        cycle. Scheduler expects only running subjobs to be reported to it.
+        """
+
+        a = {'resources_available.ncpus': 1}
+        self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
```

Comment:
I thought we decided to remove this and rather submit the jobs with exclusive flag with host=mom node to make it more generic.

Reply:
Thanks for pointing that out Anamika, I've pushed that change in